Clappr-tvOS/Classes/Base/Playback.swift

```diff
@@ -0,0 +1,131 @@
+import AVFoundation
+
+open class Playback: UIBaseObject, Plugin {
+    open class var type: PluginType { return .playback }
+
+    open class var name: String {
+        return self.init().pluginName
+    }
+
+    open var pluginName: String {
+        NSException(name: NSExceptionName(rawValue: "MissingPluginName"), reason: "Playback Plugins should always declare a name", userInfo: nil).raise()
+        return ""
+    }
+
+    open var selectedSubtitle: MediaOption?
+    open var selectedAudioSource: MediaOption?
+    fileprivate(set) open var subtitles: [MediaOption]?
+    fileprivate(set) open var audioSources: [MediaOption]?
+
+    internal(set) open var options: Options
+
+    open var source: String? {
+        return options[kSourceUrl] as? String
+    }
+
+    open var autoPlay: Bool {
```

Comment:
This is one of three differences between the shared playback, what you think about adding a pre-processor to reuse code?

Reply:
Hey @CristianDRM, I created a card to do this change in another moment.